src/frontend/src/lib/services/address.services.ts

```diff
@@ -121,20 +130,20 @@ export const loadIdbAddress = async (): Promise<{ success: boolean }> => {
 			return { success: false };
 		}
 
-		const idbEthAddress = await getIdbEthAddress(identity.getPrincipal());
+		const idbAddress = await getIdbAddress(identity.getPrincipal());
 
-		if (isNullish(idbEthAddress)) {
+		if (isNullish(idbAddress)) {
 			return { success: false };
 		}
 
-		const { address } = idbEthAddress;
+		const { address } = idbAddress;
 		addressStore.set({ tokenId, data: { data: address, certified: false } });
 
-		await updateIdbEthAddressLastUsage(identity.getPrincipal());
+		await updateIdbAddressLastUsage(identity.getPrincipal());
 	} catch (err: unknown) {
 		// We silence the error as the dapp will proceed with a standard lookup of the address.
 		console.error(
-			'Error encountered while searching for a locally stored public address in the browser.'
+			`Error encountered while searching for locally stored ${tokenId.description} public address in the browser.`
```

Comment:
i18n key in another pR?

Reply:
correct, there are a lot more around address services, address guards and address loader, so, once finished, I will do them all